app/templates/scenario-builder.html

```diff
@@ -28,7 +28,7 @@
           </div>
 
           <!-- if user selected something, show GUI -->
-          <div>
+          <div v-if="template && dataLoaded">
```

Comment:
I'll add this as a comment. At the moment, so close to delivery, I would like to refrain from rewriting the component on that level.